packages/calling/src/Metrics/index.test.ts

```diff
@@ -25,6 +26,10 @@ describe('CALLING: Metric tests', () => {
 
   beforeEach(() => {
     mockSubmitClientMetric.mockClear();
+    process.env = {
+      ...originalEnv,
+      CALLING_SDK_VERSION: MOCK_VERSION_NUMBER,
```

Comment:
Do we have a testcase where constant VERSION gets assigned instead of the version picked from process.env?

Reply:
Added the test case for this.